### ISSUE TITLE
Improve books/add-form, for existing works title not editable

### ISSUE THREAD

**Issue by bitnapper** [internetarchive/openlibrary#3794]
When adding edition from an existing work, the title-field is locked. Especially for translations the titles differ from the original work so the should be editable. The can be changed in the next from though but in many cases the editor forgets to change it than. It would be much easier if the title could be set in the first form.

### Describe the problem that you'd like solved
Make the title-fileld editable in the following form: https://openlibrary.org/books/add?work=/works/OLxxxW

### Proposal & Constraints
The field might refer to the works title when it should refer to the new editions title. It must be avoided that the works title will be changed from this form.

**Comment by anushri20** [comment]
@bitnapper Can I work on this?

**Comment by bitnapper** [comment]
> @bitnapper Can I work on this?
> 

I don't think you have to ask me. I just reported the problem.

**Comment by anushri20** [comment]
@bitnapper oh okay.

**Comment by bitnapper** [comment]
@anushri20 are you in the Slack-Group? I think everyone would be very happy, if you want to help with that problem.

**Comment by anushri20** [comment]
@bitnapper No, I am not. Please add me to the slack-Group. I would love to help you all in all possible ways.

**Comment by mekarpeles** [comment]
Hi @anushri20, I'm mek, one of the maintainers on openlibrary.org

This issue may be slightly tougher than it looks, but let's try it! Please let me know all your questions and we can work together to help you be successful in solving this issue.

Thank you!

p.s. thanks @bitnapper for facilitating + opening this issue

**Comment by anushri20** [comment]
@mekarpeles Thank you so much!! I will give it a try and I have any questions I'll definitely ask you.

**Comment by seabelis** [comment]
@anushri20 One thing to check for is that the title entered should be tied to the Edition title and not the work title. Although, the work title probably should be displayed (not editable) on the initial form. So I think the problem is really that a field is missing which is the edition title. Does this make sense?

**Comment by anushri20** [comment]
@seabelis Yes, this does make sense. I thought of doing the same thing but i am having trouble in tieing the editable title to the edition title.

Thank you.
